bucket/beekeeper-studio.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://github.com/beekeeper-studio/beekeeper-studio",
+    "description": "Cross platform SQL editor and database management app for Windows, Linux, and Mac.",
+    "version": "1.4.1",
+    "license": "MIT",
+    "url": "https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v1.4.1/Beekeeper-Studio-Setup-1.4.1.exe#/dl.7z#/dl.7z",
+    "hash": "sha512:0fbbc9963d7754c50c3bd80cae492ce2ef65f900d089668f4ea05e1b984d3ef67f281e1fca5146df1091a55d7050fdc664d5045e02e45c08b87093c9ddd716dc",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Force -Recurse"
+        ]
+    },
+    "bin": "Beekeeper Studio.exe",
```

Comment:
```suggestion
```

Reply:
we could use `& 'beekeeper studio.exe'` to launch beekeeper-studio in command line, so I think we should keep `"bin": "Beekeeper Studio.exe"` in its mainfest.

Reply:
There is no commmand line handling. Remove it.